doc/source/matdef.rst

```diff
@@ -25,12 +25,10 @@ their effect. After that, most users will probably find the :ref:`"cookbook"
 General remarks
 ---------------
 
-The strings defining Most materials in typical dgcode projects will actually be
+The strings defining most materials in typical dgcode projects will actually be
 the same strings that are used with `NCrystal (aka "NCrystal cfg-strings)
 <https://github.com/mctools/ncrystal/wiki/Using-NCrystal#uniform-material-configuration-syntax>`_,
-so you might wish to consult the NCrystal documentation for those in the
-`NCrystal wiki
-<https://github.com/mctools/ncrystal/wiki/Using-NCrystal#uniform-material-configuration-syntax>`_. Most
+so you might wish to consult the NCrystal documentation for those. Most
```

Comment:
not sure here, perhaps at least make the words "the NCrystal documentation for those" into a link?